### PR TITLE
a few minor edits to a11y best practics

### PR DESCRIPTION
**Pull request by cptvitamin**
@catong, @lamagnifica, can you take a look at these minor changes for me?

### CODE REVIEW

File: en_us/shared/accessibility/best_practices_course_content_dev.rst [hunk 1]
```diff
@@ -244,7 +247,8 @@ Consider the following examples of meaningful alt attributes for a photo of
 the famous stone bridge, Ponte Vecchio in Florence, Italy.
  
 * For a representative image of the bridge included on a page about Florence,
-  a meaningful ``alt`` attribute would be “Photo of Ponte Vecchio.”
+  a meaningful ``alt`` attribute would be “Photo of Ponte Vecchio, a famous
+  bridge and shopping center in Florence Italy.”
```
Comment (catong) [inline en_us/shared/accessibility/best_practices_course_content_dev.rst]: "Florence Italy" -> "Florence, Italy" (add comma)

File: en_us/shared/accessibility/best_practices_course_content_dev.rst [hunk 2]
```diff
@@ -720,7 +724,7 @@ Use Best Practices for Custom Content Types
 
 Using different content types in your courses can significantly add to the
 learning experience for your students. This section covers how to design
-several custom content types so that your course content is accessible all
+several custom content types so that your course content is accessible to all
 learners.
```
Comment (catong) [inline en_us/shared/accessibility/best_practices_course_content_dev.rst:729]: I can't comment on the exact line but there is a use of "alternate" on line 762 below that should be corrected to "alternative" please. Thanks!

File: en_us/shared/accessibility/best_practices_course_content_dev.rst [hunk 3]
```diff
@@ -220,13 +222,14 @@ the preferred solution.
...
 If the text contained in the image is also available in nearby HTML, or
 otherwise accessible text, including it in the ``alt`` attribute would be
-redundant. All images must include an ``alt`` attribute so you should not omit
-the ``alt`` attribute entirely. In addition, if the ``alt`` attribute is
-omitted entirely, screen readers will read the value of the ``src`` attribute
-(the path to the image on a web server) as a fallback. This is rarely helpful
-to the user and often results in a poor user experience. However, in this
-case, setting an empty ``alt=""`` attribute is acceptable. Doing so
-effectively "hides" the image from screen reader users.
+redundant. In this case, setting an empty ``alt=""`` attribute is acceptable.
```
Comment (catong) [inline en_us/shared/accessibility/best_practices_course_content_dev.rst]: I'd suggest "...setting an empty `alt` attribute (`alt=""`) is acceptable." 
Currently we are using an example as the name of the attribute.

**Comment by catong** [comment]
@cptvitamin A couple of nits and one request for an editorial correction on a line not included in this PR, then :+1:.

File: en_us/shared/accessibility/best_practices_course_content_dev.rst [hunk 4]
```diff
@@ -89,12 +89,14 @@ background, in both visual and audio elements.
 For visual elements, techniques include making sure the fonts you use are
 readable, that there is sufficient contrast between the foreground and
```
Comment (lamagnifica) [inline en_us/shared/accessibility/best_practices_course_content_dev.rst]: ...^and^ that there is...

File: en_us/shared/accessibility/best_practices_course_content_dev.rst [hunk 5]
```diff
@@ -89,12 +89,14 @@ background, in both visual and audio elements.
...
-element or conveying critical information. When images contain text, make sure
-that the text has a font size of at least 14 points and has good contrast with
-the background.  Whenever possible, use the default font and color defined in
-the edX platform, which was designed to be easy to read.  If you choose to
-override default font colors, make sure that the foreground and background
-colors have `sufficient contrast <https://leaverou.github.io/contrast-ratio/>`_.
+element or conveying critical information. If you must create images that
+contain text, make sure that the text has a font size of at least 14 points 
+and has good contrast with the background.  Images of text cannot be scaled
+or color corrected as well as HTML text. Whenever possible, use the default
```
Comment (lamagnifica) [inline en_us/shared/accessibility/best_practices_course_content_dev.rst]: as well as --> as effectively as (?) 
(this isn't wrong, it was just hard for me to parse as I read it...)

File: en_us/shared/accessibility/best_practices_course_content_dev.rst [hunk 6]
```diff
@@ -220,13 +222,14 @@ the preferred solution.
...
-the ``alt`` attribute entirely. In addition, if the ``alt`` attribute is
-omitted entirely, screen readers will read the value of the ``src`` attribute
-(the path to the image on a web server) as a fallback. This is rarely helpful
-to the user and often results in a poor user experience. However, in this
-case, setting an empty ``alt=""`` attribute is acceptable. Doing so
-effectively "hides" the image from screen reader users.
+redundant. In this case, setting an empty ``alt`` attribute (``alt=""``)
+is acceptable.  Doing so effectively "hides" the image from screen reader users.
+
+.. note:: All images *must* include an ``alt`` attribute so you should not omit
```
Comment (lamagnifica) [inline en_us/shared/accessibility/best_practices_course_content_dev.rst]: "...attribute so you should not omit..." --> "...attribute, so do not omit..."

**Comment by lamagnifica** [comment]
a few suggestions for your consideration. :+1:

**Pull request merged**